### PR TITLE
add namespace and pod name in failed event for podxxxchaos crd

### PR DESCRIPTION
**Pull request by xlgao-zju**
### Related changes

- [ ] Need to update `chaos-mesh/website`
- [ ] Need to update `Dashboard UI`
- Need to **cheery-pick to release branches**
  - [ ] release-2.1
  - [ ] release-2.0

### Checklist

CHANGELOG

- [ ] I have updated the `CHANGELOG.md`
- [ ] I have labeled this PR with "no-need-update-changelog"

Tests

- [ ] Unit test
- [x] E2E test
- [ ] No code
- [ ] Manual test (add steps below)

Side effects

- [ ] Breaking backward compatibility

### Release note 

```text
Please add a release note.

You can safely ignore this section if you don't think this PR needs a release note.
```

### DCO

If you find the DCO check fails, please run commands like below (Depends on the actual situations. For example, if the failed commit isn't the most recent) to fix it:

```shell
git commit --amend --signoff
git push --force
```

### CODE REVIEW

File: controllers/podhttpchaos/controller.go [hunk 1]
```diff
@@ -157,12 +157,14 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 		EnterNS:   true,
 	})
 	if err != nil {
+		err = errors.Errorf("fialed to apply for pod %s/%s, error: %v", pod.Namespace, pod.Name, err)
```
Comment (YangKeao) [inline controllers/podhttpchaos/controller.go:160]: How about `errors.Wrapf`?

Comment (xlgao-zju) [reply controllers/podhttpchaos/controller.go:160]: ok, will do

**Comment by xlgao-zju** [comment]
@YangKeao updated, PTAL

File: controllers/podnetworkchaos/controller.go [hunk 2]
```diff
@@ -155,6 +156,7 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 
 	err = r.SetIptables(ctx, pod, obj, pbClient)
 	if err != nil {
+		err = errors.Wrapf(err, "fialed to apply for pod %s/%s, error: %v", pod.Namespace, pod.Name)
```
Comment (WangXiangUSTC) [inline controllers/podnetworkchaos/controller.go:159]:
```suggestion
		err = errors.Wrapf(err, "fialed to apply for pod %s/%s", pod.Namespace, pod.Name)
```

File: controllers/podnetworkchaos/controller.go [hunk 3]
```diff
@@ -145,6 +145,7 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 
 	err = r.SetIPSets(ctx, pod, obj, pbClient)
 	if err != nil {
+		err = errors.Wrapf(err, "fialed to apply for pod %s/%s, error: %v", pod.Namespace, pod.Name)
```
Comment (WangXiangUSTC) [inline controllers/podnetworkchaos/controller.go:148]:
```suggestion
		err = errors.Wrapf(err, "fialed to apply for pod %s/%s", pod.Namespace, pod.Name)
```

File: controllers/podhttpchaos/controller.go [hunk 4]
```diff
@@ -157,12 +157,14 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
...
 	})
 	if err != nil {
+		err = errors.Wrapf(err, "fialed to apply for pod %s/%s", pod.Namespace, pod.Name)
 		r.Recorder.Event(obj, "Warning", "Failed", err.Error())
 		return ctrl.Result{Requeue: true}, nil
 	}
 
 	if res.StatusCode != http.StatusOK {
-		err = errors.Errorf("status(%d), apply fail: %s", res.StatusCode, res.Error)
+		err = errors.Errorf("fialed to apply for pod %s/%s, status(%d), error: %s",
```
Comment (WangXiangUSTC) [inline controllers/podhttpchaos/controller.go:166]: could update to `Wrapf` too

**Review by WangXiangUSTC — COMMENTED** [review]
rest LGTM

File: controllers/podhttpchaos/controller.go [hunk 5]
```diff
@@ -157,12 +158,15 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 		EnterNS:   true,
 	})
 	if err != nil {
+		err = errors.Wrapf(err, "fialed to apply for pod %s/%s", pod.Namespace, pod.Name)
```
Comment (WangXiangUSTC) [inline controllers/podhttpchaos/controller.go:161]: `fialed` -> `failed`

The other error messages all need to update

**Review by WangXiangUSTC — APPROVED** [review]
LGTM

**Review by cwen0 — APPROVED** [review]
LGTM

**Pull request merged**